doc/nrf/working_with_nrf/nrf52/developing.rst

```diff
@@ -34,17 +34,38 @@ You can also use FOTA updates to replace the application.
 FOTA over Bluetooth Low Energy
 ==============================
 
+FOTA updates are supported using MCUmanager's Simple Management Protocol (SMP) over Bluetooth.
+The application acts as a GATT server and allows the connected Bluetooth Central to perform the firmware update.
+Sample must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`) in order to use this feature.
+
+The application supports SMP handlers related to:
+
+* Image management.
+* Operating System (OS) management -- used to reboot the device after the firmware upload is complete.
+* Erasing settings partition -- used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
+
 To enable support for FOTA updates, do the following:
 
 * Use MCUboot as the upgradable bootloader (:kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` must be enabled).
   For more information, go to the :doc:`mcuboot:index-ncs` page.
-* Enable the mcumgr module that handles the transport protocol over Bluetooth® Low Energy as follows:
+* Enable :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option, which implies configuration of:
+
+   * All of the SMP command handlers mentioned in the previous paragraph.
+   * SMP BT reassembly feature.
+   * :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` option -- automatically extends the Bluetooth buffers, which allows to speed up the FOTA transfer over Bluetooth, but can also increase RAM usage.
```

Comment:
I would also inform that Bluetooth buffers configuration is also automatically applied to the network core child image (maybe you could mention which network core images are supported aswell - hci_rpmsg and multiprotocol_rpmsg).

Reply:
I think that we should with that for technical writer's input like @kapi-no suggested in https://github.com/nrfconnect/sdk-nrf/pull/10256#discussion_r1105847614